packages/api/src/record/models.ts

```diff
@@ -97,6 +97,18 @@ export interface ShareArchive {
   name: string;
 }
 
+export interface UpdateRecordDTO {
```

Comment:
I think I'd call this something like `RecordColumnsForUpdate`. I don't think DTO is universally understood, so best to avoid it in a name, and I'm also not sure this really is a Data Transfer Object, because it's not actually being sent over the network.